gpu/features/test/test_normals.cpp

```diff
@@ -394,6 +394,43 @@ TEST(PCL_FeaturesGPU, issue_2371)
     ne_gpu.compute(normals_gpu);
 }
 
+// See:
+// - https://github.com/PointCloudLibrary/pcl/pull/3627#discussion_r375826172
+TEST(PCL_FeaturesGPU, normals_nan_gpu)
+{
+    const std::size_t N = 5;
+
+    PointCloud<PointXYZ>::Ptr cloud(new PointCloud<PointXYZ>());
```

Comment:
This does not need to be a shared pointer.

Reply:
Fixed